static/diagrams/replays.svg

```diff
@@ -0,0 +1 @@
002 547.9204153050001 L 552.5933489775001 555.7738784300001 Z"/></g></g><g><g><rect fill="white" stroke="none" x="591.6404874556252" y="576.1928825550001" width="162.64912235625" height="24.50280495"/></g><text fill="black" stroke="none" font-family="sans-serif" font-size="11pt" font-style="normal" font-weight="normal" text-decoration="normal" x="594.4677341806253" y="593.1563629050002" text-anchor="start" dominant-baseline="alphabetic" xml:space="preserve">update("recipient", +50)</text></g><g><path fill="none" stroke="black" paint-order="fill stroke markers" d=" M 568.3002752275002 600.6956875050001 L 762.1113789050003 600.6956875050001" stroke-miterlimit="10" stroke-width="1.5706926250000002" stroke-dasharray=""/><g transform="translate(777.6298220400002,600.6956875050001) translate(-777.6298220400002,-600.6956875050001)"><path fill="black" stroke="none" paint-order="stroke fill markers" d=" M 761.9228957900002 592.8422243800002 L 777.6298220400002 600.6956875050001 L 761.9228957900002 608.5491506300001 Z"/></g></g><g><g><rect fill="white" stroke="none" x="632.3045499556252" y="628.9681547550001" width="81.32099735625" height="24.50280495"/></g><text fill="black" stroke="none" font-family="sans-serif" font-size="11pt" font-style="normal" font-weight="normal" text-decoration="normal" x="635.1317966806253" y="645.9316351050002" text-anchor="start" dominant-baseline="alphabetic" xml:space="preserve">return("ok")</text></g><g><path fill="none" stroke="black" paint-order="fill stroke markers" d=" M 777.6298220400002 653.4709597050002 L 583.8187183625001 653.4709597050002" stroke-miterlimit="10" stroke-width="1.5706926250000002" stroke-dasharray="7.5393246000000005"/><g transform="translate(568.3002752275002,653.4709597050002) translate(-568.3002752275002,-653.4709597050002)"><path fill="black" stroke="none" paint-order="stroke fill markers" d=" M 584.0072014775002 645.6174965800002 L 568.3002752275002 653.4709597050002 L 584.0072014775002 661.3244228300001 Z"/></g></g><g><g><rect fill="white" stroke="none" x="422.97500314312515" y="681.7434269550001" width="81.32099735625" height="24.50280495"/></g><text fill="black" stroke="none" font-family="sans-serif" font-size="11pt" font-style="normal" font-weight="normal" text-decoration="normal" x="425.80224986812516" y="698.7069073050002" text-anchor="start" dominant-baseline="alphabetic" xml:space="preserve">return("ok")</text></g><g><path fill="none" stroke="black" paint-order="fill stroke markers" d=" M 568.3002752275002 706.2462319050002 L 374.4891715500001 706.2462319050002" stroke-miterlimit="10" stroke-width="1.5706926250000002" stroke-dasharray="7.5393246000000005"/><g transform="translate(358.9707284150001,706.2462319050002) translate(-358.9707284150001,-706.2462319050002)"><path fill="black" stroke="none" paint-order="stroke fill markers" d=" M 374.6776546650001 698.3927687800002 L 358.9707284150001 706.2462319050002 L 374.6776546650001 714.0996950300001 Z"/></g></g></g><g/><g/><g/><g/></g></svg>
```

Comment:
Where is this used?

Reply:
Agreed version control for the win, but you can just keep in a branch on your fork